MODULE.bazel

```diff
@@ -732,6 +732,25 @@ dev_maven.install(
     ],
 )
 
+
+dev_maven.install(
+    name = "root_module_can_override",
+    artifacts = ["com.squareup:javapoet:1.11.1"],
+)
+
+bazel_dep(name = "transitive_module_can_override", version = "0.0.0")
```

Comment:
This should be marked as a `dev_dependency` so that consumers of us don't try and look up `transitive_module_can_override` in the BCR.